app/controllers/invites_controller.rb

```diff
@@ -1,10 +1,15 @@
 class InvitesController < ApplicationController
 
   def create
-    @invite = Invite.new(invite_params)
     @event = Event.find(params[:invite][:event_id])
     @room = @event.room
     @campus = @event.campus
+    @invite = Invite.find_by(user_id: current_user.id, event_id: @event.id)
+    if @invite.nil?
+       @invite = Invite.new(invite_params.merge(user_id: current_user.id))
```

Comment:
There a method in rails called `find_or_create_by` to help avoid having to write this sort of if statement. It might be helpful here 😁  http://apidock.com/rails/v4.0.2/ActiveRecord/Relation/find_or_create_by